cityofaustin/atd-moped
# Follow a project upon creation

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/9457

## Testing
**URL to test:** 

https://9457-follow-project--atd-moped-main.netlify.app/moped/projects

**Steps to test:**
Create a new project. 
Navigate to the Dashboard, projects you are following. The new project should be automatically there. 
The blue ribbon button to follow a project should be solid on the project summary view. 

I also added Rebecca to the seed data in this PR. 

---
#### Ship list
